### ISSUE TITLE
[Adaptive Card iOS - Table.FirstRowAsHeaders.json]: Voice Over is conveying the incorrect role as 'Heading' for the table headers. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#185]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 17.5.1
Application: Adaptive Cards
App Version: Version 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**

1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.5" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Table.FirstRowAsHeaders.json.Jsons' and activate it.
7. The Card will open.
8. Navigate to the table header and observe the issue.

**Actual Result:**
While navigating to the table header, screen reader is conveying the incorrect role as 'Heading'. It is announcing as 'Header heading text field group double tap to edit'.

**Expected Result:**
Voice Over should not convey the incorrect role for the table header. It should announce as ' Column Header Row 1 of 3 Column 1 of 2 ' also it should announce the row and column information.

**Note:**
This issue is repro throughout the application having table.

**User Impact:**
It will impact the screen reader user as they will not be able to know whether they are navigating to the table or heading.

**WCAG Reference:**
https://www.w3.org/TR/WCAG21/#name-role-value

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/a4660bbf-2823-4a6c-bdf6-de99d98b5d2e

### Card JSON

```JSON
NA
```
